bip-0174.mediawiki

```diff
@@ -659,6 +659,17 @@ required for aggregation. If sorting was done, then the keys must be in the sort
 | 0, 2
 | [[bip-0373.mediawiki|373]]
 |-
+| BIP 353 DNSSEC proof
+| <tt>PSBT_OUT_DNSSEC_PROOF = 0x35</tt>
+| None
+| No key data
+| <tt><RFC 9102-formatted DNSSEC Proof></tt>
+| An RFC 9102 DNSSEC `Authentication Chain Data` without the `ExtSupportLifetime` field (i.e. a series of RFC 9102 `AuthenticationChain`s) providing a DNSSEC proof to a BIP 353 DNS TXT record.
```

Comment:
backticks don't render as code formatting, if that was what you were trying to do. Otherwise, I'd suggest using single quotes here instead of backticks.